package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "HTTPS=true react-app-rewired start",
+    "start": "set HTTPS=true && react-app-rewired start",
```

Comment:
Is `yarn start` works correctly?

Reply:
`set HTTPS=true && react-app-rewired start` is working on windows, 
`HTTPS=true react-app-rewired start` is working on other operating systems such as Ubuntu.

Reply:
On macOS I still meet the following Error after `yarn start`:
```
yarn run v1.22.11
$ set HTTPS=true && react-app-rewired start

There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "babel-loader": "8.1.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-loader was detected higher up in the tree:

  /Users/yashablack/projects/ergodex/ergo-dex-frontend/node_modules/babel-loader (version: 8.2.2) 

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "babel-loader" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /Users/yashablack/projects/ergodex/ergo-dex-frontend/node_modules/babel-loader is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls babel-loader in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed babel-loader.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
you need to make `.env` file.

the content is like following :
SKIP_PREFLIGHT_CHECK=true

Reply:
Meet another one Error:
```
Failed to compile.

Failed to load plugin '@typescript-eslint' declared in '.eslintrc.json': Cannot find module '/Users/username/projects/ergodex/ergo-dex-frontend/node_modules/@typescript-eslint/eslint-plugin/node_modules/@typescript-eslint/types/dist/index.js'. Please verify that the package.json has a valid "main" entry
Referenced from: /Users/username/projects/ergodex/ergo-dex-frontend/.eslintrc.json
```